tests/test_cell_metadata.py

```diff
@@ -12,24 +12,19 @@ def test_chunk_subdocuments(self):
         self.cell_metadata = CellMetadata(
             '../tests/data/test_chunking_cell_metadata.csv', 'SCP1', '1234abc'
         )
-        # Read and transform file to data model
-        while True:
-            row = self.cell_metadata.extract()
-            if row is None:
-                break
-            self.cell_metadata.transform(row)
+        self.cell_metadata.preproccess()
         # Captures print statements from chunk_subdocuments
         captured_output = io.StringIO()
         sys.stdout = captured_output
-        for subdoc in self.cell_metadata.chunk_subdocuments(
-            'doc_name', 'fake/path/cell_metadata', 'ClusterID'
-        ):
-            pass
-
+        for cellmetadatadatamodel in self.cell_metadata.transform():
```

Comment:
We should use snake case for consistency:

```suggestion
        for cellmetadata_data_model in self.cell_metadata.transform():
```

or, following my suggestion to rename the `DataModel` class to simply `Model`:

```suggestion
        for cellmetadata_model in self.cell_metadata.transform():
```

And similarly for the later reference to this variable.